keyboards/handwired/stef9998/split_5x7/readme.md

```diff
@@ -0,0 +1,20 @@
+# Split_5x7
+
+![Split_5x7](https://i.imgur.com/fVw8al1h.jpg)
+
+A handwired Split keyboard with the unusual layout of 5x7 on either side.
+
+3d print files will be made available soon. 
+
+* Keyboard Maintainer: [Stefan](https://github.com/stef9998)
+* Hardware Supported: Pro Micro
+
+Make example for this keyboard(after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```